src/components/pagination/pagination.ts

```diff
@@ -286,3 +288,23 @@ export class MdPagination implements AfterContentInit, AfterViewInit {
     this.service.change(this.name, this.model);
   }
 }
+
+const PAGINATION_DIRECTIVES = [
+    MdPagination,
+    MdPaginationControls,
+    MdPaginationItemsPerPage,
+    MdPaginationRange
+]
+
+@NgModule({
+  providers: [
+    PaginationService // not sure about this, provider may need to be added to the components themselves and exported separately
```

Comment:
The PaginationService is a local-implementation provider for page change events.